docs/conf.py

```diff
@@ -29,11 +29,15 @@
 # The full version, including alpha/beta/rc tags
 version = "unknown"
 version_major_minor = version
-# load and overwrite version info from deepsparse package
-exec(open(os.path.join(os.pardir, "src", "deepsparse", "version.py")).read())
+
+# load and overwrite version and release info from deepsparse package
+version_path = os.path.join(os.pardir, "src", "deepsparse", "generated_version.py")
+if not os.path.exists(version_path):
+    version_path = os.path.join(os.pardir, "src", "deepsparse", "version.py")
+exec(open(version_path).read())
 release = version
```

Comment:
is this variable supposed to be is_release or release?

Reply:
this is supposed to be release, it's a special param for the sphinx build process